flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -87,6 +101,17 @@ function setup_kubernetes_for_linux {
     sudo sysctl fs.protected_regular=0
 }
 
+function download_circtl {
+    local arch=$(get_arch)
+    local crictl_version crictl_archive arch
+    crictl_version="v1.24.2"
+    crictl_archive="crictl-$crictl_version-linux-${arch}.tar.gz"
+    download_circtl_url="https://github.com/kubernetes-sigs/cri-tools/releases/download/${crictl_version}/${crictl_archive}"
+
+    wget -nv ${download_circtl_url}
+    return $?
```

Comment:
The return call is obsolete here. The function would fail if the previous `wget` fails.